features/admin.connections.v1/components/authenticators/push/quick-start.tsx

```diff
@@ -0,0 +1,175 @@
+ */
+
+import ApplicationSelectionModal from "@wso2is/admin.applications.v1/components/application-selection-modal";
+import {
+    VerticalStepper,
+    VerticalStepperStepInterface
+} from "@wso2is/admin.core.v1/components/vertical-stepper/vertical-stepper";
+import { FeatureConfigInterface } from "@wso2is/admin.core.v1/models";
+import { AppState } from "@wso2is/admin.core.v1/store";
+import { hasRequiredScopes } from "@wso2is/core/helpers";
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { GenericIcon, Heading, Link, PageHeader, Text } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useMemo, useState } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { Grid } from "semantic-ui-react";
+import BuildLoginFlowIllustration from "./assets/build-login-flow.png";
+
+/**
+ * Prop types of the component.
+ */
+type PushAuthQuickStartPropsInterface = IdentifiableComponentInterface;
+
+/**
+ * Quick start content for the Push authenticator.
+ *
+ * @param props - Props injected into the component.
+ *
+ * @returns Push Authenticator Quick start component.
+ */
+const PushAuthQuickStart: FunctionComponent<PushAuthQuickStartPropsInterface> = (
+    props: PushAuthQuickStartPropsInterface
+): ReactElement => {
+
+    const {
+        [ "data-componentid" ]: componentId
+    } = props;
+
+    const { t } = useTranslation();
+
+    const [ showApplicationModal, setShowApplicationModal ] = useState<boolean>(false);
+
+    const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state.config.ui.features);
+    const allowedScopes: string = useSelector((state: AppState) => state?.auth?.allowedScopes);
+
+    const isApplicationReadAccessAllowed: boolean = useMemo(() => (
+        hasRequiredScopes(
+            featureConfig?.applications, featureConfig?.applications?.scopes?.read, allowedScopes)
+    ), [ featureConfig, allowedScopes ]);
+
+    /**
+     * Vertical Stepper steps.
+     * @returns An array of Vertical Stepper steps.
+     */
+    const steps: VerticalStepperStepInterface[] = [
+        {
+            stepContent: (
+                <>
+                    <Text>
+                        <Trans
+                            i18nKey={
+                                "extensions:develop.identityProviders.pushAuth." +
+                                "quickStart.steps.selectApplication.content"
+                            }
+                        >
+                            Choose the { isApplicationReadAccessAllowed ? (
+                                <Link external={ false } onClick={ () => setShowApplicationModal(true) }>
+                                application </Link>) : "application" }
+                            for which you want to set up Push Authentication login.
+                        </Trans>
+                    </Text>
+                </>
```

Comment:
Unnecessary fragments.

```suggestion
                    <Text>
                        <Trans
                            i18nKey={
                                "extensions:develop.identityProviders.pushAuth." +
                                "quickStart.steps.selectApplication.content"
                            }
                        >
                            Choose the { isApplicationReadAccessAllowed ? (
                                <Link external={ false } onClick={ () => setShowApplicationModal(true) }>
                                application </Link>) : "application" }
                            for which you want to set up Push Authentication login.
                        </Trans>
                    </Text>
```